diff -pruN netgen-4.9.13-sources/configure netgen-4.9.13-2/configure
--- netgen-4.9.13-sources/configure	2015-08-24 12:15:29.560190846 +0200
+++ netgen-4.9.13-2/configure	2015-08-25 11:39:07.485726666 +0200
@@ -17837,7 +17837,7 @@ esac
 
 
 
-ac_config_files="$ac_config_files Makefile libsrc/Makefile libsrc/csg/Makefile libsrc/general/Makefile libsrc/geom2d/Makefile libsrc/gprim/Makefile libsrc/include/Makefile libsrc/interface/Makefile libsrc/linalg/Makefile libsrc/meshing/Makefile libsrc/occ/Makefile libsrc/stlgeom/Makefile libsrc/visualization/Makefile ng/Makefile nglib/Makefile tutorials/Makefile doc/Makefile windows/Makefile"
+ac_config_files="$ac_config_files Makefile libsrc/Makefile libsrc/csg/Makefile libsrc/general/Makefile libsrc/geom2d/Makefile libsrc/gprim/Makefile libsrc/include/Makefile libsrc/interface/Makefile libsrc/linalg/Makefile libsrc/meshing/Makefile libsrc/occ/Makefile libsrc/stlgeom/Makefile libsrc/visualization/Makefile nglib/Makefile "
 
 
 cat >confcache <<\_ACEOF
@@ -18905,7 +18905,8 @@ $debug ||
 if test -n "$CONFIG_FILES"; then
 
 
-ac_cr=''
+ac_cr='
+'
 ac_cs_awk_cr=`$AWK 'BEGIN { print "a\rb" }' </dev/null 2>/dev/null`
 if test "$ac_cs_awk_cr" = "a${ac_cr}b"; then
   ac_cs_awk_cr='\\r'
diff -pruN netgen-4.9.13-sources/configure.ac netgen-4.9.13-2/configure.ac
--- netgen-4.9.13-sources/configure.ac	2015-08-24 12:15:29.560190846 +0200
+++ netgen-4.9.13-2/configure.ac	2015-08-25 10:48:08.236469725 +0200
@@ -72,11 +72,11 @@ fi
 
 
 
-AC_ARG_WITH([togl], 
-        [AC_HELP_STRING([--with-togl=dir],[directory containing libTogl1.7])],
-        [togldir=$withval]
-        [togllibfl="-L$withval$"]
-        )
+#AC_ARG_WITH([togl], 
+#        [AC_HELP_STRING([--with-togl=dir],[directory containing libTogl1.7])],
+#        [togldir=$withval]
+#        [togllibfl="-L$withval$"]
+#        )
 #        [togllibfl="-L$(TK_BIN_DIR)/Togl1.7"]
 #        [togllibfl="-L/usr/local/lib/Togl1.7"]
 
@@ -124,7 +124,7 @@ AM_CONDITIONAL([NGLIB], [test x$nglibon
 AC_SUBST([TOGLLIBDIR], ["$togllibfl"])
 
 AC_CHECK_HEADER(pthread.h)
-AC_CHECK_HEADER([togl.h])
+#AC_CHECK_HEADER([togl.h])
 AC_CHECK_HEADER([GL/gl.h])
 
 AC_CHECK_FUNCS([pow])
@@ -171,13 +171,7 @@ AC_SUBST(LIBGLU)
 
 
 
-AC_CONFIG_FILES(Makefile libsrc/Makefile libsrc/csg/Makefile 
-libsrc/general/Makefile  libsrc/geom2d/Makefile libsrc/gprim/Makefile
-libsrc/include/Makefile libsrc/interface/Makefile 
-libsrc/linalg/Makefile libsrc/meshing/Makefile libsrc/occ/Makefile 
-  libsrc/stlgeom/Makefile
-  libsrc/visualization/Makefile ng/Makefile nglib/Makefile
-  tutorials/Makefile doc/Makefile windows/Makefile )
+AC_CONFIG_FILES(Makefile nglib/Makefile)
 
 AC_OUTPUT
 
@@ -208,4 +202,4 @@ AC_MSG_RESULT([
 
   Example programs will be built but not installed.
 ------------------------------------------------------------------------
-])
\ Pas de fin de ligne à la fin du fichier
+])
diff -pruN netgen-4.9.13-sources/Makefile.am netgen-4.9.13-2/Makefile.am
--- netgen-4.9.13-sources/Makefile.am	2015-08-24 12:15:29.564190971 +0200
+++ netgen-4.9.13-2/Makefile.am	2015-08-25 11:36:11.111872678 +0200
@@ -2,7 +2,7 @@ ACLOCAL_AMFLAGS = -I m4
 
 METASOURCES = AUTO
 
-SUBDIRS = libsrc ng nglib tutorials doc windows
+SUBDIRS = libsrc nglib 
 
 # TESTS = ng/netgen -batchmode
 
diff -pruN netgen-4.9.13-sources/Makefile.in netgen-4.9.13-2/Makefile.in
--- netgen-4.9.13-sources/Makefile.in	2015-08-24 12:15:29.564190971 +0200
+++ netgen-4.9.13-2/Makefile.in	2015-08-25 11:46:41.932811373 +0200
@@ -238,7 +238,7 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
 METASOURCES = AUTO
-SUBDIRS = libsrc ng nglib tutorials doc windows
+SUBDIRS = libsrc nglib
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
 
